src/components/Button/Button.less

```diff
@@ -17,6 +18,10 @@
 	line-height: @Button-size-lineHeight;
 	padding: 0 @Button-size-padding;
 
+	&[disabled] {
```

Comment:
yes and no.  There was an issue with 

```button[disabled], html input[disabled] {
    cursor: default;
}```

from `http://yui.yahooapis.com/pure/0.6.0/pure-min.css` which is our normalize css.

So I needed to add that so it would override that selector.